### ISSUE TITLE
Does it fix SST (Single Stream Transport) Limitations on MacBook Pro 15" (late 2013)

### ISSUE THREAD

**Issue by AlexanderProd** [Floris497/mac-pixel-clock-patch-V2#29]
Can I use this fix to run SST displays in 4K at 60hz on my late 2013 MBP 15" ? By standard only MST displays are supported, or SST displays at 30hz.

**Comment by Floris497** [comment]
I think you can! I own a SST 4k monitor, and it can do 4k@60Hz. my mac mini 2011 can do 4k@40Hz so it must not be a problem for your macbook.

**Comment by AlexanderProd** [comment]
Nice, thank you I'll try the patch. What are you using to run the 4K SST monitor at 4K 60hz?

**Comment by Floris497** [comment]
I'm using an external GTX 670 connected via thunderbolt to get it to 60Hz, but I think your mac will be able to do it on it's own.

Good luck!

**Comment by tomislav** [comment]
@Floris497 How are you running a GTX 670 via thunderbolt? An external enclosure?

**Comment by Floris497** [comment]
Yes, OWC helios

**Comment by AlexanderProd** [comment]
I got a SST only 4K display and I can run it on 4K @ 60hz without having to install the patch. Using a Mini DisplayPort to DisplayPort Cable. Seems like Apples support page hasn't been updated.

**Comment by Floris497** [comment]
That's good :)

**Comment by est4tic** [comment]
@AlexanderProd what monitor? I'm asking because I can't run my DELL P2715Qt at 4k 60hz with my late 2013 iMac. :(

**Comment by AlexanderProd** [comment]
@est4tic I'm using the LG 27UD68-W with my MBP 15" (late 2013).

**Comment by est4tic** [comment]
@AlexanderProd thanks, I guess I made a bad call when I got the DELL.

**Comment by corwynlund** [comment]
@AlexanderProd Has the your LG 27UD68-W continued to work seamlessly with your MBP 15" (late 2013) at 4k 60hz? I'm considering the same monitor for the same MBP, or the newer LG 27UD88-W. I'm getting conflicting reports about when this newer unit will work at 4k 60hz.

**Comment by AlexanderProd** [comment]
@corwynlund Yeah its still working fine, never had any problems with my MBP @ 4K 60Hz

**Comment by megawebspace** [comment]
hey @AlexanderProd thanks for your information. Do you have a MBP with additional graphic card or build-in Intel Iris Pro? Just want to be sure as I'm having a MBP 15" (late 2013) with build-in graphics and consider buying an LG 27UD88. Thanks for your answer.

**Comment by AlexanderProd** [comment]
@megawebspace I've got the one with build-in Iris Pro graphics, everything works fine.

**Comment by corwynlund** [comment]
@AlexanderProd and @megawebspace
The Graphics spec on my late-2013 MBP Retina is: NVIDIA GeForce GT 750M 2048 MB
My Software is: OS X 10.9.5 (13F1911)
Could either of these be a hurdle to running an LG-27UD68 monitor at 4k 60hz??
You advice is very much appreciated :)

**Comment by AlexanderProd** [comment]
@corwynlund From the hardware side no problem, but 10.9.5 could be a problem. I can't say if it'll work or not but I believe that apple changed something with SST on later version.

**Comment by corwynlund** [comment]
@AlexanderProd Great to hear the hardware not an issue. I'll have an LG-27UD68 next week. If there is a problem I'll upgrade to a newer OS. It will probably mean having to pay for a newer version of Parallels, but other that could it disrupt anything else?

**Comment by AlexanderProd** [comment]
@corwynlund Perfect. Upgrading to a new OS shouldn't brake anything, but I'd simply do a Time Machine Backup if you haven't yet.

**Comment by alissongodoi** [comment]
hi @AlexanderProd @corwynlund. I have a MacBook Pro (Retina, 15-inch, Late 2013).
I am thinking about getting the 27UD68-W (I believe is the same as 27UD68; however it is white).
Should I be concerned about compatibility? Do you guys have it working without any patches? Should I install this patch?
Thanks.

**Comment by AlexanderProd** [comment]
@alissongodoi Hi, yeah everything should work fine. You don't have to install any patches, as long as you have at least OS X Yosemite.

**Comment by corwynlund** [comment]
@alissongodoi <https://github.com/alissongodoi> I bought the 27UD68-W and
it works perfect, straight out of the box, with the same MBP (Retina,
15-inch, Late 2013) running OSX 10.9.5.

**Comment by DerekYoungMT** [comment]
@AlexanderProd Hello. I have a 2014-mid 13'' mbp, but it can't drive the 27UD68 at 60hz. I've tried to connect them both through hdmi and mini dp to dp cables. Both of them can only run as high as 36hz. Anything high than that the screen just turn black. Am I missing something? Please help me.

**Comment by AlexanderProd** [comment]
@DerekYoungMT Not sure abou this, but I assume it’s because you’re using the 13“ Modell.

**Comment by kimmoo** [comment]
@AlexanderProd Hi, I have the same macbook as you. Does the Iris graphics have enough processing power to drive the 4k comfortable? Does it increase the fan rpm significantly?

Thank you already for your previous comments. It has been surprisingly difficult to find information about this macbook + 4k display combo.

**Comment by corwynlund** [comment]
I have the same setup, and yes, the fan increases some of the time but is
liveable. I'm happy with this setup.
